packages/fiona/meta.yaml

```diff
@@ -1,8 +1,6 @@
 package:
   name: fiona
   version: 1.9.5
-  # Fails with CppException std::invalid_argument: stoi: no conversion
```

Comment:
This just works? I guess maybe there was a regression in one Emscripten update and then another update fixed it.

Reply:
Yep, I think it was a regression in LLVM, and it was fixed in another update.